llvm/test/CodeGen/WinEH/wineh-empty-seh-scope.ll

```diff
@@ -0,0 +1,14 @@
+; RUN: llc -mtriple=x86_64-pc-windows-msvc19.41.34120 %s
```

Comment:
Add FileCheck and generated checks with update_llc_test_checks

Reply:
Still prefer tests with checks to tests that just compile

Reply:
Didn't touch this yet. Never used FileCheck nor that script so I'll need to read the docs.

Reply:
Just add | FileCheck %s and then run update_llc_test_checks on the file